mbtools/generate_quiz_csv.py

```diff
@@ -0,0 +1,140 @@
+import argparse
+from collections import defaultdict
+import os
+import csv
+from mbtools import utils
+from pathlib import Path
+
+
+def write_quiz_csvs(
+    quizzes_csv, quiz_questions_csv, quiz_multichoice_answers_csv, output_path
+):
+    with open(output_path / "quizzes.csv", "w") as outfile:
+        writer = csv.writer(outfile)
+        writer = csv.writer(outfile)
+        writer.writerow(quizzes_csv.keys())
+        writer.writerows(zip(*quizzes_csv.values()))
+
+    with open(output_path / "quiz_questions.csv", "w") as outfile:
+        writer = csv.writer(outfile)
+        writer = csv.writer(outfile)
+        writer.writerow(quiz_questions_csv.keys())
+        writer.writerows(zip(*quiz_questions_csv.values()))
+
+    with open(output_path / "quiz_multichoice_answers.csv", "w") as outfile:
+        writer = csv.writer(outfile)
+        writer = csv.writer(outfile)
+        writer.writerow(quiz_multichoice_answers_csv.keys())
+        writer.writerows(zip(*quiz_multichoice_answers_csv.values()))
+
+
+def order_questions(quiz):
+    # Organize Questions by Order
+    pages = defaultdict(list)
+    for question in quiz.quiz_questions:
+        pages[question.page].append(question)
+    for page in pages.values():
+        page.sort(key=lambda h: h.slot)
+    ordered_questions = []
+    ordered_keys = list(pages.keys())
+    ordered_keys.sort()
+    for key in ordered_keys:
+        ordered_questions.extend(pages[key])
+    return ordered_questions
+
+
+def generate_quiz_data(mbz_path, output_path):
+    question_bank = utils.parse_moodle_questionbank(mbz_path)
```

Comment:
I was thinking it would be consistent with our current import configuration. I thought none of our commands actually imported mbtools/models, but now I'm seeing some of them do. I'm ok with either implementation.

Reply:
It's minor either way. Generally we were putting thing into `utils.py` if there were multiple callers (either within the package or some script and a test). In this case I don't think tests use it, so it would shave a few lines of code to just one-line it in the script and remove the helper from `utils.py` :man_shrugging: .